guest/rust/examples/ui/nodegraph/src/client.rs

```diff
@@ -0,0 +1,23 @@
+use ambient_api::{
+    components::core::{
+        layout::{
+            align_horizontal_center, align_vertical_center, height, space_between_items, width,
+        },
+        rect::{
+            background_color, border_color, border_radius, border_thickness, line_from, line_to,
+            line_width,
+        },
+        transform::translation,
+    },
+    prelude::*,
+};
+
+#[element_component]
+fn App(_hooks: &mut Hooks) -> Element {
+    Graph::el()
```

Comment:
What does this `Graph` spawn? Or is it an empty graph?

Reply:
Additionally, how do you interact with the graph and get values out of it?

Reply:
Might be a good idea to e.g. make a calculator using this, so you can see how it works

Reply:
> Additionally, how do you interact with the graph and get values out of it?

Good question. I haven't got any idea yet on how to store the data structure. Still drafting this

Reply:
> What does this `Graph` spawn? Or is it an empty graph?

It's just temporary. Might be good to change it to `GraphEditor` or something else, and perhaps this whole thing should be put into the `editor` folder.